pandas/guide/source/pandas_setup.rst

```diff
@@ -79,12 +83,31 @@ TODO
 Besides the Python `.py` files, pandas source code includes C/Cython files
 which need to be compiled in order to run the development version of pandas.
 
+For Window Users: you'll need to install the compiler toolset. Depending on which version of Python you care about, you will need to choose a different download.
+
++----------------+------------------------------------------------------------+
+|   Python       |                       You will need                        |
+|   Version      |                                                            |
++================+============================================================+
+| 3.5 and later  | Install Visual Studio 2017, select the Python development  |
+|                | workload and the Native development tools option           |
+|                | <https://www.visualstudio.com/>                            |
++----------------+------------------------------------------------------------+
+| 3.3 and 3.4    | Windows SDK for Windows 7 and .NET 4.0                     |
+|                | <https://www.microsoft.com/download/details.aspx?id=8279>  |
++----------------+------------------------------------------------------------+
+| 2.6 to 3.2     | Microsoft Visual C++ Compiler for Python 2.7               |
+|                | <https://www.microsoft.com/download/details.aspx?id=44266> |
++----------------+------------------------------------------------------------+
+
 To compile these files simply run:
     | ``cd <pandas-dir>``
     | ``python setup.py build_ext --inplace``
 
 The process will take several minutes.
 
+(For Window users: run the above commends in Anaconda Prompt)
```

Comment:
Same as before, I'd try to tell them users to use Anaconda Prompt before the commands, not after.